dashboard.tf

```diff
@@ -108,6 +108,41 @@ resource "aws_cloudwatch_dashboard" "system_health" {
         "view": "timeSeries",
         "stacked": false
       }
+    },
+    {
+      "type": "metric",
+      "x": 12,
+      "y": 3,
```

Comment:
I'm not sure about the x, y here, it looks different from any of the widgets on the Grafana docs?